lib/omni_auth/multi_provider/handler.rb

```diff
@@ -2,23 +2,26 @@ module OmniAuth
   module MultiProvider
     class Handler
       attr_reader :path_prefix, :provider_instance_path_regex, :request_path_regex,
-                  :callback_path_regex, :provider_path_prefix,
```

Comment:
I removed `provider_path_prefix` as it was only used in a couple of places and just duplicated `path_prefix` which made it a little hard to see what was going on in this class. I can put it back though if you think it is important